infrastructure/modules/cdn-frontdoor-profile/variables.tf

```diff
@@ -0,0 +1,60 @@
+variable "identity" {
```

Comment:
Can we add a description? I believe it's used to access key vault or access azure origins